app/models/activity.rb

```diff
@@ -88,7 +92,7 @@ def update_observations?
 
   def update_observations
     if update_observations?
```

Comment:
update_observations? is only true if happened_on or description has changed. Need to check updated_by here too. See spec in activities_spec.rb which tests these callbacks, they will need adding to too

```
  def update_observations?
    happened_on_previously_changed? || description_previously_changed?
  end

```